2012-08-23 18:56:54,747 UserViewIndex     line:35   ERROR    CreateUserIndex
Traceback (most recent call last):
  File "/home/nitin/projects/tx2/tx2/Users/Views/UserViewIndex.py", line 33, in CreateUserIndex
    return render_to_response('TXtemplates/UserSystem/User/Register.html',{},context_instance=RequestContext(HttpRequest))
  File "/usr/local/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 20, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 181, in render_to_string
    t = get_template(template_name)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 157, in get_template
    template, origin = find_template(template_name)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 138, in find_template
    raise TemplateDoesNotExist(name)
TemplateDoesNotExist: TXtemplates/UserSystem/User/Register.html
2012-08-23 19:00:18,241 UserViewIndex     line:35   ERROR    CreateUserIndex
Traceback (most recent call last):
  File "/home/nitin/projects/tx2/tx2/Users/Views/UserViewIndex.py", line 33, in CreateUserIndex
    return render_to_response('TXtemplates/UserSystem/User/Register.html',{},context_instance=RequestContext(HttpRequest))
  File "/usr/local/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 20, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 181, in render_to_string
    t = get_template(template_name)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 157, in get_template
    template, origin = find_template(template_name)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 138, in find_template
    raise TemplateDoesNotExist(name)
TemplateDoesNotExist: TXtemplates/UserSystem/User/Register.html
2012-08-23 19:01:04,477 UserViewIndex     line:35   ERROR    CreateUserIndex
Traceback (most recent call last):
  File "/home/nitin/projects/tx2/tx2/Users/Views/UserViewIndex.py", line 33, in CreateUserIndex
    return render_to_response('TXtemplates/UserSystem/User/Register.html',{},context_instance=RequestContext(HttpRequest))
  File "/usr/local/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 20, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 181, in render_to_string
    t = get_template(template_name)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 157, in get_template
    template, origin = find_template(template_name)
  File "/usr/local/lib/python2.7/site-packages/django/template/loader.py", line 138, in find_template
    raise TemplateDoesNotExist(name)
TemplateDoesNotExist: TXtemplates/UserSystem/User/Register.html
